db/seeds.rb

```diff
@@ -1,7 +1,341 @@
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Editor.create(
+  kind: "board",
+  title: "Editor-in-Chief",
+  first_name: "Arfon",
+  last_name: "Smith",
+  login: "arfon",
+  email: "",
+  avatar_url: "http://joss.theoj.org/assets/editors/arfon-33e4ac87ed2e073216fda0d3383b25fc3a15cc7ac5e70aab3e9f85f68b3d4a65.jpg",
+  categories: [],
+  url: "http://arfon.org/",
+  description: <<-STR.strip_heredoc()
+    A lapsed academic with a passion for new models of scientific collaboration, he's used big telescopes
+    to study <a href="http://www.arfon.org/thesis" target="_blank">dust in space</a>, built
+    sequencing pipelines in <a href="http://www.sanger.ac.uk/" target="_blank">Cambridge</a> and
+    engaged millions of people in online citizen science by co-founding the
+    <a href="http://www.zooniverse.org" target="_blank">Zooniverse</a>.
+  STR
+)
+
+Editor.create(
+  kind: "topic",
+  first_name: "Lorena A",
+  last_name: "Barba",
+  login: "labarba",
+  email: "",
+  avatar_url: "http://joss.theoj.org/assets/editors/lorena-ac22212e9dd478fc2190ab583cd2f12f6870e25e5666641381bb1c9463fc366d.png",
+  categories: ["Computational Science and Engineering", "High-performance Computing"],
+  url: "http://lorenabarba.com/",
+  description: <<-STR.strip_heredoc()
+    Associate Professor of Mechanical and Aerospace Engineering at the George Washington University,
+    leading a research group in computational fluid dynamics, computational physics and
+    high-performance computing. Member of the Board for NumFOCUS, a non-profit in support of
+    open-source scientific software.
+  STR
+)
+
+Editor.create(
+  kind: "topic",
+  first_name: "Jason",
+  last_name: "Clark",
+  login: "jasonclark",
+  email: "",
+  avatar_url: "http://joss.theoj.org/assets/editors/jason-4aca9e8821dd02f8ea26b8fe499cccddd9758707f9a3d711ecc7a71e6a76f6af.jpg",
+  categories: ["Information Sciences", "Semantic Web"],
+  url: "http://www.jasonclark.info/",
+  description: <<-STR.strip_heredoc()
+    Associate Professor, Librarian, and Head of Archival Informatics and Special Collections at
+    Montana State University (MSU) Library, specializing in software development, metadata and data
+    modeling, linked and structured data, search engine optimization, and interface design. You can
+    find him on ORCID at
+    <a href="http://orcid.org/0000-0002-3588-6257" target="_blank">http://orcid.org/0000-0002-3588-6257</a>
+    and as <a href="https://twitter.com/jaclark" target="_blank">@jaclark on Twitter</a>.
+  STR
+)
+
+Editor.create(
+  kind: "topic",
+  first_name: "George",
+  last_name: "Githinji",
+  login: "biorelated",
+  email: "",
+  avatar_url: "http://joss.theoj.org/assets/editors/george-d24f98577bcc40a745b81f7ee5c4b6a3e5f150b563f940cb792ba92f88ed6483.jpg",
```

Comment:
@parrish - what should we do about URLs like this? How does encoding the asset pipeline stuff (i.e. `george-d24f98577bcc40a745b81f7ee5c4b6a3e5f150b563f940cb792ba92f88ed6483.png` work in the long run?

Reply:
I guess perhaps it doesn't matter as long as the file doesn't change? http://guides.rubyonrails.org/asset_pipeline.html#what-is-fingerprinting-and-why-should-i-care-questionmark